engines/scumm/script_v5.cpp

```diff
@@ -2875,6 +2875,11 @@ void ScummEngine_v5::printPatchedMI1CannibalString(int textSlot, const byte *ptr
 "Oooh, that's nice.\xFF\x03"
 "Simple.  Just like one of mine.\xFF\x03"
 "And little.  Like mine.";
+	} else if (strncmp((const char *)ptr, "/LH.ITA/", 8) == 0) {
+		msg =
+"Oooh, che bello.\xFF\x03"
+"Semplice.  Proprio come uno dei miei.\xFF\x03"
+"E piccolo.  Come il mio.";
```

Comment:
Could you please fix the indentation here, together with the English strings above?
These should be indented one level after msg (so, two levels to the right of the if block)

Reply:
Sure! I indented it like that only because the english one was approved like that :P